dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/IpUtils.java

```diff
@@ -49,12 +49,11 @@ public static String longToIp(long ipLong) {
     ipNumbers[2] = ipLong >> 8 & tmp;
     ipNumbers[3] = ipLong & tmp;
 
-    StringBuilder sb = new StringBuilder(16);
-    sb.append(ipNumbers[0]).append(DOT)
-            .append(ipNumbers[1]).append(DOT)
-            .append(ipNumbers[2]).append(DOT)
-            .append(ipNumbers[3]);
-    return sb.toString();
+    String sb = ipNumbers[0] + DOT +
```

Comment:
why not StringBuilder?

Reply:
> why not StringBuilder?

The functions of the two are the same, but String+ looks more concise, and from a performance perspective, String is better

Reply:
Because the use here is the same, and String performance is better in this case, the code is also more concise

Reply:
In this case ,I think it's hard to say which one is better,  you might be right.